packages/@jsii/spec/package.json

```diff
@@ -31,11 +31,14 @@
     "package": "package-js"
   },
   "dependencies": {
-    "jsonschema": "^1.4.0"
+    "jsonschema": "^1.4.0",
+    "sort-json": "^2.0.0",
+    "fs-extra": "^9.1.0"
```

Comment:
Not a big fan of adding `fs-extra` as a dependency. I'd like to keep dependencies minimal if possible.

`fs-extra` doesn't really deliver anything that the regular `fs` module doesn't deliver (starting at Node 12 or so).

Instead, do the following:

```ts
import { promises as fs } from 'fs';

// ...

await fs.writeFile('filename', data, { encoding: 'utf-8' });
```

This uses the [fs.promises](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_promises_api) API, which is also async (just like `fs-extra`).